lib/Flux.js

```diff
@@ -31,6 +35,11 @@ function createMatcher({ keys, re }) {
  * @extends Error
  */
```

Comment:
Enrichir le commentaire en ajoutant: ... when a path can't be matched with an action